pkg/apis/triggers/v1alpha1/trigger_binding_validation.go

```diff
@@ -30,11 +30,29 @@ func (t *TriggerBinding) Validate(ctx context.Context) *apis.FieldError {
 
 // Validate TriggerBindingSpec.
 func (s *TriggerBindingSpec) Validate(ctx context.Context) *apis.FieldError {
-	return validateParameters(s.Params)
+	if err := validateInputParams(s.InputParams); err != nil {
+		return err
+	}
+	if err := validateOutputParams(s.OutputParams); err != nil {
+		return err
+	}
+	return nil
+}
+
+func validateInputParams(params []v1alpha1.ParamSpec) *apis.FieldError {
```

Comment:
This _could_ be recycled for validating input and output?

Reply:
I created a helper function to use for both